src/test/java/pages/LogInPage.java

```diff
@@ -13,7 +13,7 @@ public LogInPage(WebDriver driver) {
         this.driver = driver;
     }
 
-    @FindBy(xpath = "//*[text() = 'Sign In']")
+    @FindBy(xpath = "//*[@id=\"root\"]/main/div/div[1]/div/span/h6")
```

Comment:
Hi! This xpath (xpath = "//*[@id=\"root\"]/main/div/div[1]/div/span/h6") does not find needed web-element on Linux. Using possibility of Chrome to create xpath, I mean Copy - Copy XPath, 
xpath has to be like this
(xpath = "//*[@id="root"]/main/div/div[1]/div/span/h6").

And maybe more correct to write xpath in this way:
//*[text() = 'to continue E-Restaurant']